### DISCUSSION TITLE
Frax Funds not arrived at Solana add - pending for days at "Awaiting completion on Solana"

### DISCUSSION THREAD

**Discussion by thomascochrane** [wormhole-foundation/wormhole#216] [Problems / bugs] [answered]
Hi,

I attempted to transfer Frax using the Solana Wormhole and the transaction got stuck on "Awaiting completion on Solana" the funds were removed from my ETH wallet and were not deposited in my Solana Wallet.

Here is the ETH transaction: 
https://etherscan.io/tx/0x937eaa88c51ed8198732453dd67985553a538cd21da4a620cf9a13ab687a10a2

Here is the Solana transaction: 
https://explorer.solana.com/tx/42KJWj7vnLbwpVwVdKM5tUiNvGLmMVseikQePsTdoLyZRDJqCtbJYbywY5ZVgzJTzBytvBjD5EQnVgFGcMeghctz

Solana Wallet Address:
59htrnEc4dQUZJ3SxxMwMbjNX5dxzicL69zTdKKNZSBc

Could you please resolve this bridge transaction?

Please let me know if you need any additional information.

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/4GKWqomAVV3HU27fZPt9RvwbuAcw9wAYT9yG38FhHbYj3zLVTBeyWTM6aPUXEVqo2PQY9Hpiq9BWbJFx3MKEYzYT
